skipruntime-ts/ts/examples/sum.ts

```diff
@@ -37,11 +38,12 @@ class Add implements Mapper<string, TJSON, string, TJSON> {
     it: NonEmptyIterator<TJSON>,
   ): Iterable<[string, TJSON]> {
     const v = it.first() as number;
-    const ev = this.other.maybeGetOne(key) as number;
+    const ev = this.other.maybeGetOne(key) as Opt<number>;
```

Comment:
You can definitely use tighter types: replace the `TJSON` in this class by `number`, and `inputCollections` in `reactiveCompute` by `inputCollections: Record<string, EagerCollection<string, number>>,` or even `: { input1: EagerCollection<string, number>, input2: ... }`, it will make `tsc` (unsoundly) happy.